mypy_django_plugin/transformers/functional.py

```diff
@@ -1,12 +1,17 @@
+from typing import Any
+
 from mypy.checkmember import analyze_member_access
 from mypy.errorcodes import ATTR_DEFINED
 from mypy.nodes import CallExpr, MemberExpr
 from mypy.plugin import AttributeContext
 from mypy.types import AnyType, Instance, TypeOfAny
 from mypy.types import Type as MypyType
+from mypy.version import __version__ as mypy_version
 
 from mypy_django_plugin.lib import helpers
 
+mypy_version_info = tuple(map(int, mypy_version.partition("+")[0].split(".")))
```

Comment:
Switched to a version-based check. This is somewhat ugly, but I didn't want to add `packaging` as a dependency.